ERROR - 2025-03-01 12:05:43 --> Error connecting to the database: mysqli_sql_exception: Accès refusé pour l'utilisateur: 'root'@'@localhost' (mot de passe: NON) in C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'nouvelle_vague', 3306, '', 0)
#1 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Models\Monmodele.php(101): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...', Array)
#4 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Controllers\Controleurmain.php(38): App\Models\Monmodele->getUserById('7')
#5 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(942): App\Controllers\Controleurmain->pgProfil()
#6 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
#7 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp64\www\NouvelleVague\NouvelleVague\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Accès refusé pour l'utilisateur: '****'@'@localhost' (mot de passe: NON) in C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Models\Monmodele.php(101): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...', Array)
#3 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Controllers\Controleurmain.php(38): App\Models\Monmodele->getUserById('7')
#4 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(942): App\Controllers\Controleurmain->pgProfil()
#5 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
#6 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp64\www\NouvelleVague\NouvelleVague\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2025-03-01 12:05:43 --> Unable to connect to the database.
Main connection [MySQLi]: Accès refusé pour l'utilisateur: '****'@'@localhost' (mot de passe: NON)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\Monmodele.php(101): CodeIgniter\Database\BaseConnection->query('SELECT * FROM utilisateur WHERE id = ?', [...])
 3 APPPATH\Controllers\Controleurmain.php(38): App\Models\Monmodele->getUserById('7')
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgProfil()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 12:52:19 --> Invalid file: "tempsForts.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('tempsForts.php')
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('tempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('tempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 12:52:41 --> Invalid file: "tempsForts.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('tempsForts.php')
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('tempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('tempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 12:53:05 --> Invalid file: "tempsForts.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('tempsForts.php')
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('tempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('tempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 14:18:05 --> syntax error, unexpected token ")", expecting "->" or "?->" or "{" or "["
in APPPATH\Views\pgTempsForts.php on line 19.
 1 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('pgTempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('pgTempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 14:18:09 --> syntax error, unexpected token ")", expecting "->" or "?->" or "{" or "["
in APPPATH\Views\pgTempsForts.php on line 19.
 1 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('pgTempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('pgTempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 14:18:20 --> syntax error, unexpected token ")", expecting "->" or "?->" or "{" or "["
in APPPATH\Views\pgTempsForts.php on line 19.
 1 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('pgTempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('pgTempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 14:19:14 --> syntax error, unexpected token ")", expecting "->" or "?->" or "{" or "["
in APPPATH\Views\pgTempsForts.php on line 19.
 1 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('pgTempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('pgTempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 14:24:02 --> syntax error, unexpected token ")", expecting "->" or "?->" or "{" or "["
in APPPATH\Views\pgTempsForts.php on line 20.
 1 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('pgTempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('pgTempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 15:41:54 --> Undefined variable $user_id
in APPPATH\Views\pgTempsForts.php on line 29.
 1 APPPATH\Views\pgTempsForts.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user_id', 'C:\\wamp64\\www\\NouvelleVague\\NouvelleVague\\app\\Views\\pgTempsForts.php', 29)
 2 SYSTEMPATH\View\View.php(228): include('C:\\wamp64\\www\\NouvelleVague\\NouvelleVague\\app\\Views\\pgTempsForts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('pgTempsForts', [], true)
 5 APPPATH\Controllers\Controleurmain.php(65): view('pgTempsForts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 17:58:15 --> Invalid file: "pgTempsForts.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('pgTempsForts.php')
 2 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('pgTempsForts', [], true)
 3 APPPATH\Controllers\Controleurmain.php(65): view('pgTempsForts', [...])
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->pgTempsForts()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-03-01 17:59:33 --> Cannot redeclare App\Controllers\Controleurmain::inscriptionTempsFort()
in APPPATH\Controllers\Controleurmain.php on line 180.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-03-01 17:59:58 --> Cannot redeclare App\Controllers\Controleurmain::inscriptionTempsFort()
in APPPATH\Controllers\Controleurmain.php on line 180.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-03-01 18:00:01 --> Cannot redeclare App\Controllers\Controleurmain::inscriptionTempsFort()
in APPPATH\Controllers\Controleurmain.php on line 180.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-03-01 18:00:18 --> Cannot redeclare App\Controllers\Controleurmain::inscriptionTempsFort()
in APPPATH\Controllers\Controleurmain.php on line 180.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-03-01 18:00:25 --> Cannot redeclare App\Controllers\Controleurmain::inscriptionTempsFort()
in APPPATH\Controllers\Controleurmain.php on line 180.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-03-01 18:00:25 --> Cannot redeclare App\Controllers\Controleurmain::inscriptionTempsFort()
in APPPATH\Controllers\Controleurmain.php on line 180.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-03-01 18:00:36 --> Cannot redeclare App\Controllers\Controleurmain::inscriptionTempsFort()
in APPPATH\Controllers\Controleurmain.php on line 180.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-03-01 18:01:57 --> Cannot redeclare App\Controllers\Controleurmain::inscriptionTempsFort()
in APPPATH\Controllers\Controleurmain.php on line 180.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2025-03-01 18:39:28 --> Error connecting to the database: mysqli_sql_exception: Accès refusé pour l'utilisateur: 'root'@'@localhost' (mot de passe: OUI) in C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'nouvelle_vague', 3306, '', 0)
#1 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Models\Monmodele.php(79): CodeIgniter\Database\BaseConnection->query('SELECT * FROM t...')
#4 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Controllers\Controleurmain.php(12): App\Models\Monmodele->getTempsForts()
#5 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(942): App\Controllers\Controleurmain->index()
#6 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
#7 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp64\www\NouvelleVague\NouvelleVague\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Accès refusé pour l'utilisateur: '****'@'@localhost' (mot de passe: OUI) in C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Models\Monmodele.php(79): CodeIgniter\Database\BaseConnection->query('SELECT * FROM t...')
#3 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Controllers\Controleurmain.php(12): App\Models\Monmodele->getTempsForts()
#4 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(942): App\Controllers\Controleurmain->index()
#5 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
#6 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp64\www\NouvelleVague\NouvelleVague\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2025-03-01 18:39:28 --> Unable to connect to the database.
Main connection [MySQLi]: Accès refusé pour l'utilisateur: '****'@'@localhost' (mot de passe: OUI)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\Monmodele.php(79): CodeIgniter\Database\BaseConnection->query('SELECT * FROM temps_fort')
 3 APPPATH\Controllers\Controleurmain.php(12): App\Models\Monmodele->getTempsForts()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->index()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-03-01 18:50:26 --> mysqli_sql_exception: La table 'nouvelle_vague.tempsforts' n'existe pas in C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\MySQLi\Connection.php(306): mysqli->query('DELETE FROM `te...', 0)
#1 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `te...')
#2 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `te...')
#3 C:\wamp64\www\NouvelleVague\NouvelleVague\system\Database\BaseBuilder.php(2790): CodeIgniter\Database\BaseConnection->query('DELETE FROM `te...', Array, false)
#4 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Models\Monmodele.php(159): CodeIgniter\Database\BaseBuilder->delete()
#5 C:\wamp64\www\NouvelleVague\NouvelleVague\app\Controllers\Controleurmain.php(307): App\Models\Monmodele->deleteTempsFort('3')
#6 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(942): App\Controllers\Controleurmain->supprimerTempsFort('3')
#7 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
#8 C:\wamp64\www\NouvelleVague\NouvelleVague\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\wamp64\www\NouvelleVague\NouvelleVague\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2025-03-01 18:50:26 --> La table 'nouvelle_vague.tempsforts' n'existe pas
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2790): CodeIgniter\Database\BaseConnection->query('DELETE FROM `tempsforts`
WHERE `id` = :id:', [...], false)
 2 APPPATH\Models\Monmodele.php(159): CodeIgniter\Database\BaseBuilder->delete()
 3 APPPATH\Controllers\Controleurmain.php(307): App\Models\Monmodele->deleteTempsFort('3')
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Controleurmain->supprimerTempsFort('3')
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Controleurmain))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
